### ISSUE TITLE
How do i delete limiter as well as limiter key in redis after 5 min - cluster

### ISSUE THREAD

**Issue by gsmtamil** [SGrondin/bottleneck#66]
My Requirement is : The clustered limiter lifetime is only 5 min, after 5 min i dont want to execute the limiter. at the same time the limiter key has to delete from redis.
I have tried using group in this I am not able to achieve cluster concept.

**Comment by SGrondin** [comment]
The Group will be automatically deleted from Redis after 5 minutes of inactivity, even if your program has exited. Bottleneck automatically sets an [expirations](https://redis.io/commands/expire) on all the Redis keys it uses. Why do you need to clear them at the same time as you stop using the limiter?

**Comment by gsmtamil** [comment]
I have tested multiple times using cluster the Redis key is not deleting after 5 min.
I have waited 15 min also to check it is not deleting so that I have requested extra option to set manually like configuration.

**Comment by SGrondin** [comment]
If you are still having this issue, what version of Bottleneck are you using?

**Comment by gsmtamil** [comment]
Yes this issue is still available
I have tested with 
redis version: 4.0.9
bottleneck version: 2.6.0
redis npm version: 2.8.0
limiter config
`const createLimiter = ({ id, maxConcurrent = 1, minTime = 500, clearDatastore = false }) => {
  return new Bottleneck({
    maxConcurrent,
    minTime,
    id,
    datastore: "redis",
    clearDatastore,
    clientOptions: {
      host: redis.host, //host: "127.0.0.1",
      port: redis.port, // port: 6379
      password: redis.password
    }
  });
};`

Example : 
id: ABC1
id: ABC2
each id will hit limiter.schedule()   8 times from different node cluster, i need to enable some option based on first come first serve, the functionality is woking fine. But the cluster id is not removing from redis

if i run this command KEYS *ABC* in redis client the output is:
 1) "b_ABC1_settings"
 2) "b_ABC1_running"
 3) "b_ABC2_settings"
 4) "b_ABC2_running"

after 15 min also the keys is available in redis:

Right now I have done this fix:
whenever limiter is calling that time i am setting manually expires 5min using "b_"+id+"_setting" and "b_"+id+"_running" this keys

**Comment by SGrondin** [comment]
Thank you for reporting this! 🎉 

I did some experiments and you're right that right now only the limiters created from a Group are expired. It will be fixed in v2.7.0, all the keys will be expired and the expiration time will be configurable.

**Comment by SGrondin** [comment]
v2.7.0 has just been released, you'll be able to do:
```js
const createLimiter = ({ id, maxConcurrent = 1, minTime = 500, clearDatastore = false }) => {
  return new Bottleneck({
    maxConcurrent,
    minTime,
    id,
    datastore: "redis",
    clearDatastore,
    timeout: 300000, // 5 minutes
    clientOptions: {
      host: redis.host,
      port: redis.port,
      password: redis.password
    }
  });
};
```

Please let me know if it didn't fix it for you.
